### ISSUE TITLE
Stop showing retweets!!!!

### ISSUE THREAD

**Issue by Beckman** [cezarsa/silver_bird#355]
I've turned OFF retweets from this user, but silver brid keeps showing and he is retweeing like mad. Please respect twitters OFF setting for retweets.

**Comment by ghost** [comment]
Yes please support that. I have an account that I really like to follow for what they say but the constant promotional retweet are REALLY annoying.

Thanks!
